src/main/java/rx/internal/schedulers/NewThreadWorker.java

```diff
@@ -48,8 +49,17 @@
     static {
         EXECUTORS = new ConcurrentHashMap<ScheduledThreadPoolExecutor, ScheduledThreadPoolExecutor>();
         PURGE = new AtomicReference<ScheduledExecutorService>();
-        PURGE_FORCE = Boolean.getBoolean(PURGE_FORCE_KEY);
         PURGE_FREQUENCY = Integer.getInteger(FREQUENCY_KEY, 1000);
+
+        // Forces the use of purge even if setRemoveOnCancelPolicy is available
+        final boolean purgeForce = Boolean.getBoolean(PURGE_FORCE_KEY); // Compiler should inline this
```

Comment:
I don't understand what this trailing comment is telling me. Why is it here? Speculation about compiler optimizations make for bad comments. Especially since this is a local field in a static block that's only ever run once where perf doesn't _really_ matter.